core/src/test/scala/unit/kafka/admin/TimeConversionTests.scala

```diff
@@ -0,0 +1,54 @@
+/**
+ * Licensed to the Apache Software Foundation (ASF) under one or more contributor license agreements. See the NOTICE
+ * file distributed with this work for additional information regarding copyright ownership. The ASF licenses this file
+ * to You under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with the
+ * License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on
+ * an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations under the License.
+ */
+package kafka.admin
+
+import java.text.{ParseException, SimpleDateFormat}
+import java.util.Date
+
+import org.junit.Assert.fail
+import org.junit.Test
+
+class TimeConversionTests {
```

Comment:
Since this is specific to ConsumerGroupCommand, can we include that in the name somewhere?

Reply:
thanks for reviews!

Instead of having an new test file, I move the test case to ResetConsumerGroupOffsetTest since there is only one test case in TimeConversionTest.

Reply:
I had put it in a separate class because it had no need for the overhead that comes from the kafka server test harness.

Reply:
I had put it in a separate class because it had no need for the overhead that comes from the kafka server test harness.

Reply:
@hachikuji Got it. Let me address previous comment to avoid the overhead - "Since this is specific to ConsumerGroupCommand, can we include that in the name somewhere?"